modal.js

```diff
@@ -0,0 +1,178 @@
+const myMultipost = document.getElementById('portfolioModal');
+
+const myRecentWorks = [
```

Comment:
- [ ] [OPTIONAL] You can add more projects to the list so your project's popup is different when clicking on different projects